pkg/docker/docker.go

```diff
@@ -574,8 +574,8 @@ func (d *stiDocker) PullImage(name string) (*api.Image, error) {
 				if msg.Error != nil {
 					return msg.Error
 				}
-				if msg.ProgressMessage != "" {
-					glog.V(4).Infof("pulling image %s: %s", name, msg.ProgressMessage)
+				if msg.Progress != nil {
+					glog.V(4).Infof("pulling image %s: %s", name, msg.Progress.String())
```

Comment:
did this change the output?

Reply:
afaics the output looks the same, I can pastebin samples of before/after if you would like.

Reply:
i'll take your word for it.